sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/responses/AzureResponsesContentImageSample.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai.responses;
+
+import com.azure.ai.openai.responses.models.*;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.util.Configuration;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Base64;
+
+import static java.lang.System.exit;
+
+/**
+ * This sample demonstrates how to use the Responses Input Content Image  to generate text using the Azure OpenAI service.
+ */
+public class AzureResponsesContentImageSample {
+
+    /**
+     * Main method to run the sample.
+     * @param args Command line arguments (not used).
+     */
+    public static void main(String[] args) {
+        // Create a client
+        ResponsesClient client = new ResponsesClientBuilder()
+            .endpoint(Configuration.getGlobalConfiguration().get("AZURE_OPENAI_ENDPOINT"))
+            .credential(new AzureKeyCredential(Configuration.getGlobalConfiguration().get("AZURE_OPENAI_KEY")))
+            .serviceVersion(AzureResponsesServiceVersion.V2025_03_01_PREVIEW)
+            .httpLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+            .buildClient();
+
+        String base64Image = null;
+        try {
+            String fileName = "ms_logo.png";
+            byte[] imageBytes = Files.readAllBytes(Paths.get("src/samples/java/com/azure/ai/openai/resources/" + fileName));
+            base64Image= Base64.getEncoder().encodeToString(imageBytes);
+        } catch (IOException e) {
+            System.out.println("Exception: " + e);
+            exit(-1);
```

Comment:
Maybe it's better to simply `return;` here. I am not entirely sure if this is in violation of any sample guideline. @mssfang , Are you aware if this is what we could normally use in a sample?